src/OrchardCore.Modules/OrchardCore.Contents/Views/Content.SummaryAdmin.cshtml

```diff
@@ -82,3 +88,22 @@
         </div>
     </div>
 }
+
+
+@functions
+{
+    public bool HasHtml(IHtmlContent content)
+    {
+        if (content is HtmlString htmlString)
+        {
+            return !String.IsNullOrWhiteSpace(htmlString.Value?.Trim());
+        }
+
+        using var writer = new System.IO.StringWriter();
+
+        content.WriteTo(writer, System.Text.Encodings.Web.HtmlEncoder.Default);
```

Comment:
If you really need to render the menu:
- HtmlEncoder should be resolved by DI. (@inject HtmlEncoder HtmlEncoder)
- And get the stringbuilder from a pool, passed to the StringWriter.